dockerfiles/che-fabric8/assets/branding/product.json

```diff
@@ -5,12 +5,6 @@
     "faq": "https://www.eclipse.org/che/docs/che-7/hosted-che/#hosted-che-faq-and-troubleshooting_hosted-che"
   },
   "configuration": {
-    "menu": {
-      "disabled": ["organizations", "factories"]
-    }, 
-    "features": {
-      "disabled": ["workspaceSharing", "kubernetesNamespaceSelector"]
-    },
     "prefetch": {
       "cheCDN":"/api/cdn-support/paths"
```

Comment:
it's not implemented yet in new dashboard. There is an issue https://github.com/eclipse/che/issues/18728

Reply:
BTW should be even implement it?..
We can implement `prefetch.resources` which is common enough and cheCDN should depends on the plan if it's going to be reused like on che-dev cluster.

Reply:
I believe any approach will work as long as pre-fetching will be possible to speed up the time for lodaing. Adding @davidfestal who initially developed the CDN prefetching mechanism

Reply:
The way it currently works is the following:
- The dashboard was pointing to the `/api/cdn-support/paths` endpoint to retrieve the list of resources to prefetch.
- The `/api/cdn-support/paths` endpoint was implemented by an `rh-che` extension to:
    - extract the CDN index annotation (containing a JSON list of CDN artifacts associated to the given version of Che-Theia) from the che-theia docker image configured as the default editor
    - return it